python/cugraph/graph/c_graph.pyx

```diff
@@ -795,3 +795,72 @@ class Graph:
             del degree_col
 
         return df
+
+    def compute_degree(self, vertex_subset=None):
```

Comment:
Would "in_out_degree" or "degrees" be more consistent than "compute_degree" when comparing with the existing "in_degree", "out_degree", "degree" functions?

Reply:
@Alex you raise an interesting question.  Spark GraphX just has a structure that is created when a Graph is created with inDegreem outDegree, and degree - the use doesn't have to explicitly call a functions.   NetworkX also has separate in and out calls - which is what James currently has.  This function just get in and out in one call.  
I like "degrees" just because I don't think we need the "compute_" part

Reply:
I like calling the function 'degrees' as well. If there aren't any objections I will go ahead and change it to that.